components/sections/NavItem/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { ArrowUpRightIcon } from '@heroicons/react/24/solid';
+import classNames from 'classnames';
+import Link from 'next/link';
+import { useMemo, type FC } from 'react';
+
+import styles from './index.module.css';
+
+export enum NavItemType {
```

Comment:
Can this be a string type? `type NavItemType = 'nav' |'footer';`

Reply:
changed to string. But imho, I think using an enum would make it easier to refactor later (if needed)

Reply:
Not really, TypeScript ensures that the values follow the suitable types. Also, enums are usually a discouraged practice. As in practical terms they're not a type, but an object.